testdata/imports/compose_enums.txtar

```diff
@@ -77,19 +79,21 @@ export interface Compose {
   localstr: LocalEnum;
   localstrd: LocalEnumD;
   localstrover: LocalEnumD;
+  union: dep.DepEnumNumericD;
+  unionStrings: LocalEnum;
 }
 
 export const defaultCompose: Partial<Compose> = {
   impnum: dep.DepEnumNumeric.Three,
   impnumd: dep.DepEnumNumericD.One,
-  impnumover: dep.DepEnumNumericD.Three,
+  impnumover: 3,
   impstr: dep.DepEnum.Bar,
   impstrd: dep.DepEnumD.Foo,
-  impstrover: dep.DepEnumD.Bar,
+  impstrover: 'bar',
```

Comment:
this is the correct default output under the new rules, but in the generated interface definition `impstrover` is unchanged (line 75), it's still `dep.DepEnumD`. That has to change to `(dep.DepEnumD | 'bar')`

Reply:
Rmdr: same for `impnumover`.

Wondering if using values not present in the range of values defined within the enum would make it easier to follow & understand? Or maybe even having both examples?